webapp/src/gitjson.tsx

```diff
@@ -568,6 +568,7 @@ class GithubComponent extends data.Component<GithubProps, GithubState> {
             this.handleGithubError(e);
         } finally {
             this.hideLoading()
+            pxt.tickEvent("github.commit.success");
```

Comment:
Can you add a failure path too? Like github.commit.failure?finally will execute even though there is an exception.

Reply:
oh yes totally! I added a failure message instead of this success one, since we already have a tick event for when the commit button is pressed. I can add this back though if you think it'd be better to have the two events closer

Reply:
Good thumb of rule is, you don't want to join the two independent events to figure out the information. So just add the success event as well here. So you have to just query one thing.

Reply:
I made the change, but I don't fully understand - won't that be two events anyways?

Reply:
Sorry, I was not clear. I pushed the commit to check the success and only emit then.

Reply:
oh yeah that totally makes sense! thanks :D